news/_posts/2022-02-14-new-theme-announce.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: Hello 2022, meet our new Front office theme
+subtitle: Here are some more details about it
+date: 2022-02-14
+authors: [NeOMakinG]
+icon: icon-laptop
+tags: [news, theme, 8.x]
+---
+
+## A new front-office theme is being built 
+
+A new front-office theme is being built, for a long time, many community members have been pushing for a rework of the Classic front-office theme. The maintainer team is happy to announce that a new front-office theme is being built, and you can [find it here](https://github.com/PrestaShop/theme-refacto).
+
+## Why such a decision?
+
+The PrestaShop community provided a [lot of feedback](https://github.com/PrestaShop/PrestaShop/issues/14533) and ideas, with suggestions like:
+
+- Upgrade Bootstrap to version 5, the latest LTS
+- Rework the structure inside  `src`  directory, which was `_dev` before.
+- Remove the `core.js` file that theme developers could not modify or even remove
+- Use [lazy loading](https://en.wikipedia.org/wiki/Lazy_loading).
+- Choose, and use a consistent CSS class naming strategy, for example [BEM](https://github.com/getbem/getbem.com/). This is more maintainable and easier to read.
+- Use [JSON-LD](https://en.wikipedia.org/wiki/JSON-LD) instead of microdata attributes.
+- Add as many hooks as possible to extend the base theme.
+- Separate the PrestaShop styling from the Bootstrap raw styling.
+- Reduce the number of dependencies.
+- Do not rely on jQuery, the theme will use standard JavaScript APIs. Note that jQuery will still be available because a lot of modules need it.
+- Some of the Front office pages would refresh large parts of a page even though only a small part needed to be updated.
+- Emit as many events as possible in JavaScript in order to allow modules and themes to use them.
+- Reuse as many Bootstrap components as possible.
+
+It was impossible to fix all these issues on the Classic theme without breaking the whole ecosystem of modules and community themes. We decided that it was time for a change and a new theme. This is not a complete rework because we did not want to change everything, but we were able to significantly improve the default PrestaShop theme. This new theme can be considered a rebuilt Classic Theme, different, yet similar.
+
+## How to try the new theme? 
+
+Clone the [theme repository](https://github.com/PrestaShop/theme-refacto) into your `themes` directory, run `npm install && npm run build`, enable it from the back office, and off you go!
```

Comment:
Is it planned to rename the repository once a definitive name has been decided for it? If so it means we need this name before we publish this article right?

Reply:
We could keep this one and redirect to the new one in the readme if needed, it shouldn't be blocking